src/views/pages/helloWorldNEAR/styled.ts

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { desktopDevice, tabletDevice, mobileDevice } from 'views/theme/mediaQuery';
+
+export const BackgroundWrapper = styled.div`
+  background-color: ${({ theme }) => theme.colors.backgroundSecondary};
+  color: ${({ theme }) => theme.colors.text};
+  min-height: 100vh;
+`;
+
+export const ContentWrapper = styled.main`
+  position: absolute;
+  top: 35vh;
+  width: 100%;
+  text-align: center;
+  ${desktopDevice} {
+    top: 34vh;
+  }
+`;
+
+export const FormWrapper = styled.form`
+   display: flex;
+   justify-content: center;
+   flex-direction: column;
+   align-items: center
+   text-align: center;
+   padding: 2rem 0;
+   margin: auto;
+   ${desktopDevice} {
+      width:500px;
+   }
+`;
+
+export const SubmitButtonStyled = styled.button`
```

Comment:
use Button component. And I noticed that for form and for modal the buttons are the same. Add prop size to Button component with styles and then use it like \<Button size="large">hello world\</Button>